backend/src/database/entities/personnel/personnel.entity.ts

```diff
@@ -212,6 +212,57 @@ export class PersonnelEntity extends BaseEntity {
   })
   availabilityConfirmedOn: Date;
 
+  @Column({
+    name: 'chips_last_ping',
+    type: 'date',
+    nullable: true,
+  })
+  chipsLastPing: Date;
+
+  @Column({
+    name: 'chips_last_action_date',
+    type: 'date',
+    nullable: true,
+  })
+  chipsLastActionDate: Date;  // Action date, ie. when their CHIPS profile was last updated
+
+  @Column({
+    name: 'chips_profile_missing',
+    type: 'boolean',
+    default: false,
+  })
+  chipsProfileMissing: boolean;
+
+  @Column({
+    name: 'chips_last_updated_properties',
+    type: 'jsonb',
+    default: () => "'{}'",
+  })
+  chipsLastUpdatedProperties: { [key: string]: string | object };
```

Comment:
Should we define interfaces with the keys we will be using in the json? Just for readability

Reply:
Hmmm... could we define an interface with optional values? Reason being, if we need to query the jsonb column later, it helps to know  key names. Otherwise, maybe a comment here to explain this could be any key of personnel?